Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
So if you are ok, I would change the base image to official Docker image `java:8`. It will be an OpenJDK instead of an Oracle JDK...